pkg/runtime/framework/plugins/coscheduling/coscheduling.go

```diff
@@ -69,6 +69,8 @@ var (
 const Name = "CoScheduling"
 
 // +kubebuilder:rbac:groups=scheduling.x-k8s.io,resources=podgroups,verbs=create;get;list;watch;update;patch
+// +kubebuilder:rbac:groups=node.k8s.io,resources=runtimeclasses,verbs=get;list;watch
+// +kubebuilder:rbac:groups="",resources=limitranges,verbs=get;list;watch
```

Comment:
Does the plugin also create/delete the resources ?
https://github.com/kubeflow/trainer/blob/9c2febd9cbe3a070435cd051aabc6dd3e5c43f2a/pkg/runtime/framework/plugins/coscheduling/coscheduling.go#L232-L238